_data/gaming.yml

```
@@ -1,4 +1,10 @@
 websites:
+  - name: 2K Games
```

```suggestion
  - name: 2K
```
I only see 2K referred to by exactly this name only.